src/katt_callbacks.erl

```diff
@@ -241,7 +245,30 @@ http_request( #katt_request{ method = Method
   Hdrs1 = proplists:delete("x-katt-sleep", Hdrs0),
   Hdrs = proplists:delete("x-katt-timeout", Hdrs1),
   timer:sleep(Sleep),
-  lhttpc:request(Url, Method, Hdrs, Body, Timeout, []).
+  %% need to module-prefixed call for meck's case
+  %% http_request_now(Url, Method, Hdrs, Body, Timeout, []).
+  katt_callbacks:http_request_now(Url, Method, Hdrs, Body, Timeout, []).
+
+http_request_now(Url, Method, Hdrs, Body, Timeout, []) ->
```

Comment:
> Why not move this to katt_util?

Not a bad idea, actually: ~`katt_util:external_http_request`.
But the argumentation fails to hit home :) `*_util` shouldn't be a dump for all internal functions you want to meck.

re: lhttpc signature - unfinished PR. Obviously the signature and the tests should be cleaned up.. By who? Let me turn the wheel of fortune and I tell you right away:)
Well, the tests themselves need proper work irrespective of this PR (there must be a clean=readable=maintainable way to test katt).

re: fusco - I hope you're not arguing that one should choose fusco just because I'd keep the call signature intact :) Went with hackney in this PR due to longevity (# of commits) and popularity (# of github stars).